public/components/overview/mitre/mitre.tsx

```diff
@@ -142,7 +148,20 @@ export class Mitre extends Component {
   }
 
   render() {
-    const { isLoading } = this.state;
+    const { isLoading, notPermissions } = this.state;
+
+    if (notPermissions) {
+      return (
+        <WzEmptyPromptNoPermissions
+          permissions={[
+            { action: 'agent:read', resource: 'agent:id:*' },
+            { action: 'syscheck:read', resource: 'agent:id:*' },
+            { action: 'mitre:read', resource: '*:*:*' },
+          ]}
+        />
+      );
+    }
```

Comment:
I think it doesn't need to show the prompt using the `notPermissions` state. You could use the HOC to wrap the MITRE component:

Are `syscheck:read` and `mitre:read` action permissions needed here at the same time? I am not sure about that.